doc/onboarding.md

```diff
@@ -213,7 +213,7 @@ needs to be pointed out separately during the onboarding.
 * Optionally, include your personal pronouns.
 * Label your pull request with the `doc` and `notable-change` labels.
 * Run CI on the PR. Because the PR does not affect any code, use the
-  `node-test-pull-request-lite` CI task. Alternatively, use the usual
+  `node-test-pull-request-lite-pipeline` CI task. Alternatively, use the usual
   `node-test-pull-request` CI task and cancel it after the linter and one other
   subtask have passed.
```

Comment:
Maybe one for @nodejs/build to comment on, but my own experience with Jenkins is that cancelling jobs can sometimes leave workspaces in a bad state (usually manifesting in git checkout errors on subsequent runs) depending on where the job was at at the time it was cancelled so I'm not generally in favour of encouraging the cancellation of jobs.

Reply:
From my personal experience, a lot of git checkout errors / infra failures in general with ci.nodejs.org can be linked back to issues with the machine itself, not usually due to cancelling jobs early. Also, you could say there's a benefit from new collaborators practicing how to launch `node-test-pull-request` jobs. I'd rather folks let the build go all the way through and finish, but I don't think they're "breaking our infra" by cancelling it early.

Reply:
I'd prefer that cancelling jobs is not encouraged at all, I've seen a lot of failures because of it. It may not be the most frequent issue but it happens.

At least two cases are know: 1) Abort during a git operation will leave behind a lock file that will make all following jobs on that machine fail; 2) Visual Studio will not stop running immediately after the job is aborted, locking some files and making git clean fail for some time/jobs (happens easily when there are queued jobs). Note that Jenkins does not offer us any reliable way of running anything between jobs, so fixing this is not straightforward.

(To be clear, I'm also not blocking this PR, but would be in favor of deleting the sentence starting at "Alternatively".)

Reply:
I will make a new PR after landing to draw more attention to this change.

Reply:
PR: https://github.com/nodejs/node/pull/21977